templates/macros/actions.html.twig

```diff
@@ -258,8 +281,8 @@
             {% set actions = actions|merge({'edit': {'url': path('admin_timesheet_edit', {'id': timesheet.id}), 'class': class}}) %}
         {% endif %}
 
-        {% if not timesheet.end and is_granted('stop', timesheet) %}
-            {% set actions = actions|merge({'stop': path('admin_timesheet_stop', {'id' : timesheet.id})}) %}
+        {% if timesheet.end and is_granted('start', timesheet) %}
```

Comment:
Why would you restart a timesheet from the team view?

Reply:
@kevinpapst Would it be OK to restart from the team view if the user of the last timesheet entry is the current user? That would be useful for single user scenarios when freelancers are using kimai just for themselves (that's my case). In such a scenario there is no difference between "My times" and the team view.

Reply:
Having one more action item there doesn't hurt. I am just questioning if it is necessary here, but then "why not".